src/components/Article/ArticlesHomepage.tsx

```diff
@@ -87,6 +107,21 @@ const ArticlesHomepage: React.FC = () => {
     fetchArticles();
   }, []);
 
+  const handleArticleLinkClick = (e: any) => {
```

Comment:
Remove this.

Reply:
I added this so if an user not signed in trys to open any article he gets a message that he must sign up before viewing entire article. Do you want him to see article anyways?